src/context/json_schema_provider.ts

```diff
@@ -1,6 +1,15 @@
 import _ from "underscore";
 
-import { ContextProvider } from "./provider";
+import { AnyObject } from "../entity/in_memory";
+import { ContextProvider, ContextProviderConfig } from "./provider";
+
+interface JSONSchemaFormDataProviderConfig extends ContextProviderConfig {
+    isUsingJinjaVariables?: boolean;
+}
+
+interface UiSchema extends AnyObject {
```

Comment:
RJSF has a `UiSchema` type (see [here](https://github.com/rjsf-team/react-jsonschema-form/blob/825566f73c5fe8f74bf287d5cc1c0c2ad36734ab/packages/utils/src/types.ts#L879)), should we use that instead or do you think it is too specific?

Reply:
Done